### PR TITLE
feat: get presentations

### PR DESCRIPTION
**Pull request by acarnagey**
Able to get both sent and received presentations from the api

### CODE REVIEW

File: src/operations/presentations/getPresentations.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const headers = generateHeaders();
+  try {
+    const { data: receivedResponse } =
+      await api.presentations.getPresentationsSharedWithMe({ headers });
+    const { data: sentResponse } =
+      await api.presentations.getPresentationsSharedWithOthers({ headers });
+    const response = {
+      page: 0,
+      count: (sentResponse as any).count + (receivedResponse as any).count,
+      items: [...(receivedResponse as any).items, ...(sentResponse as any).items],
```
Comment (OR13) [inline src/operations/presentations/getPresentations.ts:17]: hmm this seems not correct.

File: src/operations/presentations/getPresentations.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import * as core from "@actions/core";
+import { Api } from "../../vdp";
+import { axiosConfig } from "../../config";
+import { generateHeaders } from "../../utils";
+
+export const getPresentations = async () => {
```
Comment (OR13) [inline src/operations/presentations/getPresentations.ts:6]: lets make this name clear that it is both sent and received.

**Review by OR13 — CHANGES_REQUESTED** [review]
This action should not merge upstream APIs, it should be 1-1 with the upstream APIs.

please implement getSentPresentations and getReceivedPresentations separately.

Also make sure to use the operation names (operationId) from OAS for the method names in this API.

**Comment by acarnagey** [comment]
Ok sounds good

**Review by OR13 — APPROVED** [review]
LGTM

**Pull request merged**